culture/review_culture.md

```diff
@@ -0,0 +1,142 @@
+> If you want to go fast, go alone. If you want to go far, go together.
+
+As a community, we value going far over going fast. To go far, we need to stay
+aligned in our values, goals, and practices. One of the ways that we do this is
+through cultivating review culture.
+
+# On Review Culture
+
+Review culture came to us from the practices of software engineers. Large
+codebases are incredibly complex: they contain the ideas and work of hundreds
+of people, manifested in millions of lines of code. They need to remain
+logically consistent, even as they grow and change. A big part of how engineers
+manage is through code review: every single change to the system is reviewed by
+at least one other person before becoming official, or "merging".
+
+Ensuring that all work is reviewed brings several benefits. Most obviously, it
+leads to higher quality work, because the final output includes feedback and
+suggestions from others. Just as importantly, reviews enable the exchange of
+knowledge, context, and perspective within the team. In a good review, both the
+contributor and the reviewer can teach each other about how work gets done on
+the project. Finally, review ensures that every part of the project is
+understood by at least two people.
+
+# The Review Process
+
+## Changes
+The review process orients around _changes_, which are the units of work being
+reviewed.
+
+A change can take many forms, for example:
+- Adding a new guide or piece of documentation
+- Adding a new module to the codebase
+- Modifying an existing guide or module
+
+
+## Participants in the Review Process
+
+### Contributor
+
+The contributor is the person proposing a change. Anyone can decide to be a
+contributor: the only requirements are that you have a change you need to
+propose, and that you agree to be a respectful participant in the review
+process. (When we have a code of conduct--link to it here.)
+
+### Reviewer
+
+The reviewers are the people who review the change, suggest changes to it, and
+eventually approve it. Anyone in the community can act as a reviewer: the only
+requirements are that you have some insight to share (or questions to ask)
+about the change, and that you agree to be a respectful participant in the
+process.
+
+### Maintainer
+
+The maintainers are responsible for the health and maintenance of the project,
+and have final say in the review process. Maintainers can block changes they
+think are harmful, or approve a change over others' objections. Someone can be
+a maintainer within a specific domain: for example, acting as maintainer for a
+particular repository, or even maintaining a particular guide. Since
+maintainers have special powers, they also have a special responsibility to act
```

Comment:
Maintainers already have a lot of power here. If we start flowing money according to scores, that power will increase. Wary of abuse, but also bottlenecks. For instance, let's say SourceCred gets big, and is installed on all sorts of different OSse, hardware, etc. And configured in all sorts of different ways. That means we'll start seeing a lot of different guides pop up (e.g. how to install SourceCred on Mac Mojave, configured for my problem). A maintainer will probably not have the bandwidth or ability to review all these. If a contributor adds an HTML link to an outside guide to the docs, does that flow cred to the guide's author? The contributor that submits the PR?

 For "core documentation", which is widely applicable, I think a maintainer makes sense.

Reply:
I think they're fair points. Though I think it's less about maintainers and more about when we should or shouldn't use a review process.

For example, contributions that don't have a review process (wiki's, forums, ...) there are also fewer appointed maintainers and when they're there, it's much more hands-off. But when it _does_ have a review process, there's likely an involved maintainer as well.

Reviews cause centralization, so we should be mindful where we apply it.